websocket/jetty12/pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+The MIT License
+
+Copyright (c) 2019, CloudBees, Inc.
```

Comment:
2024?

Reply:
not sure really. as there are just copy of previous files with only few import changes.
so I cannot see real need to change that.

Reply:
It is common practice to preserve the created year and current year in the copyright notice; exempli gratia, "Copyright (c) 2019, 2024 CloudBees, Inc."

Reply:
frankly it depends of projects.. Is it a common practice here? I'm not really sure 
I cannot see real modification of year when a file is modified here. 
looking at this commit https://github.com/jenkinsci/jenkins/commit/f3900d928ffc36dda47c5c6b0db416531e9543c1
I cannot see the copyright range modified for the pom